INSTALLATION.md

```diff
@@ -10,6 +10,7 @@ This document provides instructions on how to set up and start a running instanc
 - [Installation](#installation)
   - [Setting up this repository](#setting-up-this-repository)
   - [Setting up npm](#setting-up-npm)
+  - [Setting up Typescript](#setting-up-typescript)
```

Comment:
No need to change this file

Reply:
In the recent pull request, we added a new feature for an auto Table of Contents generator. There might be cases where some headings aren't included in the TOC, and as a result, these lines are part of this PR.